src/api/readProductTeam/src/v1/tests/test_steps.py

```diff
@@ -1,14 +1,14 @@
-from aws_lambda_powertools.utilities.data_classes import APIGatewayProxyEvent
-from hypothesis import given
-from hypothesis.strategies import builds
+# from aws_lambda_powertools.utilities.data_classes import APIGatewayProxyEvent
```

Comment:
Commented code?

Reply:
it's not totally clear to me whether or not we need this test, but will wait til PI-124 until i delete the file